pysyft/types/torch/v1/tensor.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package pysyft.types.torch.v1;
+
+message Tensor {
+    oneof id {
+        int32 id_int = 1;
+        string id_str = 2;
+    }
+    bytes bin = 3;
```

Comment:
Currently there may be 2 variants of `bin`, depending on `serializer`:
 * torch - pytorch serializer that makes binary blob. It's fast but useless for cross-language messages because it's very python-specific serialization, you need python to decode it.
 * all - makes tuple of shape, type, and array of values. This was meant as most generic way to send tensor

Reply:
Okay, cool, that confirms the suspicion I mentioned above that there’s a need for a more generic tensor representation.

Reply:
Here's how generic serialization result looks like:
https://github.com/OpenMined/PySyft/pull/2762/commits/e6063f51dd74b825760e31585ec715d5c4f27f31#diff-89f20dd40f2dbc5ece5509b332de8394R531

I suspect TF will have its own tensor type. Not sure if it's better to join everything in one tensor type with `bin` field being `oneof` or it's better to have separate per-framework tensor types and then generic tensor type for cross-language/framework messaging.

Reply:
Thanks! I was leaning toward the latter. This `Tensor` schema just covers the Torch-specific serialization; I'll add a schema to match the generic serialization shortly.

Reply:
I take that back; l see that I've conflated these two options in this schema. This is closer to the generic Tensor with various serialization options, so let's give that a try and if it doesn't work out, we can always split it out into separate types.